pkgs/by-name/ca/cargo-v5/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  dbus,
+  udev,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "cargo-v5";
+  version = "0.8.2";
+
+  src = fetchFromGitHub {
+    owner = "vexide";
+    repo = "cargo-v5";
+    rev = "9d5f6e014c80838ff2b0cca401bcbd518c1e9274";
```

Comment:
```suggestion
rustPlatform.buildRustPackage rec {
  pname = "cargo-v5";
  version = "0.8.2";

  src = fetchFromGitHub {
    owner = "vexide";
    repo = "cargo-v5";
    rev = "refs/tags/v${version}";
```

Reply:
The repository does not (as of writing) have version tags.